src/components/icon-tile/icon-tile.module.css

```diff
@@ -29,7 +29,7 @@
 .size-medium {
   --size: 40px;
   --padding: 8px;
-  --icon-size: 16px;
+  --icon-size: 24px;
```

Comment:
This was actually a bug, our icon size was off according to the [hds spec. ](https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/HDS-Product---Components?node-id=2632%3A8072)